cabal-install/src/Distribution/Client/CmdRepl.hs

```diff
@@ -338,8 +338,8 @@ addDepsToProjectTarget deps pkgId ctx =
       | packageId pkg /= pkgId = SpecificSourcePackage pkg
       | SourcePackage{..} <- pkg =
         SpecificSourcePackage $ pkg { srcpkgDescription =
-          srcpkgDescription & (\f -> L.allCondTrees $ traverseCondTreeC f)
-                            %~ (deps ++)
+          srcpkgDescription & L.traverseBuildInfos
+                            %~ over L.targetBuildDepends (deps ++)
```

Comment:
Using `over` and its operator synonym `%~` one after another is a bit confusing. Is this the same as:
```
srcpkgDescription
  &  L.traverseBuildInfos
  %  L.targetBuildDepends 
  %~ (deps ++)
```
(compose the lenses first, and apply `over` / `%~` last) ? If yes, I'd prefer that. I'm not sure my version typechecks and/or does the same thing, but you get the idea.

Reply:
Done